### ISSUE TITLE
Training loss does not decrease

### ISSUE THREAD

**Issue by BelieferQAQ** [morpheus9631/DeepLung#7]
excuse me？ In the training stage, the loss of my training set and verification set are basically the same. Have you encountered this problem

**Comment by morpheus9631** [comment]
I use 10-fold Cross-validation to train my model.
Hope it will be helpful to you
